lib/ASTGen/Sources/ASTGen/PluginHost.swift

```diff
@@ -258,34 +258,34 @@ class PluginDiagnosticsEngine {
 
     // Emit the diagnostic
     var mutableMessage = message
-    let diag = mutableMessage.withUTF8 { messageBuffer in
-      SwiftDiagnostic_create(
+    let diag = mutableMessage.withBridgedString { bridgedMessage in
+      Diagnostic_create(
         cxxDiagnosticEngine, bridgedSeverity,
-        cxxSourceLocation(at: position),
-        messageBuffer.baseAddress, messageBuffer.count)
+        bridgedSourceLoc(at: position),
+        bridgedMessage)
```

Comment:
Note to myself: We should also run `swift-format` on the `ASTGen` sources, I think.

Reply:
Yeah, that'd be nice.

Reply:
> Note to myself: We should also run `swift-format` on the `ASTGen` sources, I think.

I’ve been doing some progress on `ASTGen` the past few days as well and I figured to borrow the `.swift-format` file from `swift-syntax` with a line length adjusted to 100. Do we want it to be 80 as in the Standard Library?

Reply:
Just for the record: I very much like not having a maximum line length in swift-syntax because in my experience the line length limitation sometimes causes us choose less descriptive, shorter variable names. And in my opinion it hasn’t lead to excessively long lines.

Also, just to make an informed decision, here’s a chart of the line lengths in non-generated SwiftSyntax code.

<img width="980" alt="Screenshot 2023-05-26 at 10 31 06" src="https://github.com/apple/swift/assets/4062178/977e0eb1-54fc-4655-97e6-8ccfc011891b">

Some key data points:
- ≤160 characters: 99.9%
- ≤150 characters: 99.8%
- ≤140 characters: 99.7%
- ≤130 characters: 99.5%
- ≤120 characters: 99.1%
- ≤110 characters: 98.6%
- ≤100 characters: 97.9%
- ≤90 characters: 96.7%
- ≤80 characters: 94.4%

Reply:
> Also, I think that the format (and thus the maximum line length) should be consistent between swift-syntax and ASTGen.

Sure, I generally think having a single format across swift-syntax and Swift parts of the compiler would be great. 

> I just did some tests on swift-syntax and all changes for the line length to 160 seemed reasonable to me. Dropping it to 150 already gave me a couple cases where I preferred the previous formatting. The trend continue as I dropped it further.

I guess it has to be a compromise based on what line length reasonably fits on most of our screens without wrapping. Either way I do believe we need some kind of limit below 200 that we can use and refer to in reviews or else this is bound to get out of control. Personally, I probably wouldn’t be fine with anything more than 160.

Reply:
Something is better than nothing, so if that's 160 I'm personally fine with that :)